tensorflow-mixin/dashboards/tensorflow-overview.libsonnet

```
@@ -695,7 +709,7 @@ local containerLogsPanel = {
           template.new(
             'model_name',
             promDatasource,
-            'label_values(:tensorflow:serving:request_count{job=~"$job",instance=~"$instance"}, model_name)',
+            'label_values(:tensorflow:serving:request_count{%(tensorflowSelector)s}}, model_name)', % $._config,
```

Do not need second last comma at the end
```suggestion
            'label_values(:tensorflow:serving:request_count{%(tensorflowSelector)s}}, model_name)' % $._config,
```